### PR TITLE
Refresh hello dragon image and text

### PR DESCRIPTION
**Pull request by petros**
The latest DRGTK version default output has different text and also has a rotating DragonRuby image. This commit updates image and text.

For the image, I tried to use the same specs as the original image, but please let me know if it needs to change.

For the text, I added the fact that the image rotates. But, again, if leaving this out was intentional, please let me know. 😃🙇‍♂️

PS: Thank you so much for writing this book. I always wanted to see something like this for DRGTK.

### CODE REVIEW

File: src/img/hello-dragon.jpg
Comment (brettchalupa) [inline src/img/hello-dragon.jpg:1]: The image in the book shows "Hello Dragon" to illustrate changing text from the default. Could you update the screenshot you've changed to include that text change that the book shows?

**Review by brettchalupa — CHANGES_REQUESTED** [review]
Thank you so much for this contribution! The moving target nature of the default boilerplate may cause the book to become more and more outdated the more it changes, so keeping it up to date via your PR is quite helpful.

In addition to my image change request, would you be willing to update the default code too, in this file? https://github.com/DragonRidersUnite/book/blob/ae908153c3425306b759f2b119d0fbfd4ea382af/src/code/chapter_01/01_default/app/main.rb

Thanks again, @petros!

File: src/img/hello-dragon.jpg
Comment (petros) [reply src/img/hello-dragon.jpg:1]: Oh my Zeus! I am sorry. I was shortsighted. Didn't notice there were two images (before and after) and code involved. I am going to tidy things up, and let you know 😄.

**Comment by petros** [comment]
@brettchalupa I took the initiative and I updated the two images, the code, but also adjusted the text in Chapter 01 to align with the code. I had to introduce the concept of a hash because the code introduces that, but I am not sure if I have found the right balance. Please let me know if you would like to rephrase or tweak my changed text in any way, and I will do it. 🙇

File: src/img/hello-dragon.jpg
Comment (petros) [reply src/img/hello-dragon.jpg:1]: I think I have fixed the mess with the images 😄

**Review by brettchalupa — APPROVED** [review]
Great job, thank you so much! I really appreciate the contribution and you making these more comprehensive changes.

**Pull request merged**